easybuild/easyconfigs/c/CLHEP/CLHEP-2.4.1.0-foss-2017b.eb

```diff
@@ -0,0 +1,24 @@
+easyblock = 'CMakeMake'
+
+name = 'CLHEP'
+version = '2.4.1.0'
+
+homepage = 'http://proj-clhep.web.cern.ch/proj-clhep/'
+description = """The CLHEP project is intended to be a set of HEP-specific foundation and
+ utility classes such as random generators, physics vectors, geometry and linear algebra.
+ CLHEP is structured in a set of packages independent of any external package."""
+
+toolchain = {'name': 'foss', 'version': '2017b'}
+toolchainopts = {'pic': True}
+
+source_urls = ['http://proj-clhep.web.cern.ch/proj-clhep/DISTRIBUTION/tarFiles/']
+sources = [SOURCELOWER_TGZ]
+checksums = ['d14736eb5c3d21f86ce831dc1afcf03d423825b35c84deb6f8fd16773528c54d']
+
+builddependencies = [('CMake', '3.9.5')]
```

Comment:
Preferably use the same version of CMake that is already in use for CLHEP-2.4.0.0-intel-2017b.eb, i.e. CMake 3.10.0
since this is also a 2017b toolchain.

Reply:
changed the cmake version.